src/utils/getFilteredTodos.ts

```diff
@@ -0,0 +1,14 @@
+import { Todo } from '../types/todo/Todo';
+import { StatusFilter } from '../enums/statusFilter';
+
+export function getFilteredTodos(todos: Todo[], statusFilter: StatusFilter) {
+  switch (statusFilter) {
+    case StatusFilter.Active:
+      return [...todos].filter(todo => !todo.completed);
+    case StatusFilter.Completed:
+      return [...todos].filter(todo => todo.completed);
+    case StatusFilter.All:
+    default:
+      return [...todos];
```

Comment:
DRY
can we make smth like [...todos].filter(todo => switch(filter) case: active return todo => !todo.completed ....